modules/lead/sdm/operator-toolchain-values.tpl

```diff
@@ -22,6 +22,12 @@ product:
   rbac:
     serviceAccountAnnotations: ${product_service_account_annotations}
 
+aws-event-mapper:
+  enabled: ${aws_event_mapper_enabled}
```

Comment:
the it looks like the variable you're passing to the `template_file` is `enable_aws_event_mapper`.  either this line or that variable will have to be changed.